src/convertToObject.js

```diff
@@ -1,18 +1,18 @@
 'use strict';
 
-/**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
   // write your code here
-}
+  const result = {};
+  const sourceArray = sourceString.split(';');
+
+  sourceArray.map(elem => {
+    const smallArray = elem.split(':');
 
+    if (smallArray.length > 1) {
+      result[smallArray[0].trim()] = smallArray[1].trim();
+    }
+  });
+
+  return result;
```

Comment:
Code style: There is an unnecessary empty line before the 'return' statement. It's good practice to keep your code concise and avoid unnecessary blank lines.

Reply:
fixed